packages/react-jsx-highmaps/build/use-lodash-es.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        var source = path.node.source
+        source.value = source.value.replace(/^lodash($|\/)/, 'lodash-es$1')
+      },
+    },
+  }
+}
```

Comment:
As there are no commonjs modules published in any package, we could get rid of this and lodash dependency by just using lodash-es in source code. That would simplify things.

Reply:
It could be changed in all packages.

Reply:
Would this mean we could also remove the need for lodash-webpack-plugin?

Reply:
babel-plugin-lodash could be then removed. 

lodash-webpack-plugin removes some inner parts from lodash itself, so it is still useful.

Reply:
I can do it, if you don't mind?

Reply:
No I don't mind at all, thanks!